### PR TITLE
Fix #67

### LINKED ISSUE #67: SecurityContextInterface is deprecated since Symfony 2.6
This issue is a reminder to replace SecurityContextInterface with corresponding alternatives in the following two classes:
 ```Gos\Bundle\WebSocketBundle\Client\Auth\WebsocketAuthenticationProvider```
 ```Gos\Bundle\WebSocketBundle\Event\ClientEventListener```

Security context is deprecated since 2.6:
http://symfony.com/blog/new-in-symfony-2-6-security-component-improvements

Meanwhile, the use of the bundle causes two warnings in the profiler:
screen shot 2015-10-09 at 20 25 25

### PR DESCRIPTION
**Pull request by piotrantosik**
Fix #67.
In ```ClientEventListener``` security is not used and i remove it.

### CODE REVIEW

File: DependencyInjection/GosWebSocketExtension.php [hunk 1]
```diff
@@ -30,6 +30,18 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $configs = $this->processConfiguration($configuration, $configs);
 
+        // Set the SecurityContext for Symfony <2.6
+        // Should go back to simple configuration after drop <2.6 support
+        if (interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
```
Comment (ProPheT777) [inline DependencyInjection/GosWebSocketExtension.php]: If we are sure about symfony version for this feature it's better to directly use the `VERSION_ID` provided by the kernel. What do you think ?

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/Kernel.php#L64

**Comment by ProPheT777** [comment]
:+1: 

I will try it as soon as possible, thank you @piotrantosik

File: DependencyInjection/GosWebSocketExtension.php (hunk 1 above)
Comment (piotrantosik) [inline DependencyInjection/GosWebSocketExtension.php]: You're right is better option, I changed.

Comment (piotrantosik) [inline DependencyInjection/GosWebSocketExtension.php]: @ProPheT777 see this comment - https://github.com/symfony/symfony-docs/issues/5879#issuecomment-155122336
previous version with ```interface_exist``` is better choice.
In specific case ```HttpKernel```  component version may be different than ```Security```

**Pull request closed without merging**